server/src/main/java/org/apache/iotdb/db/consensus/statemachine/DataRegionStateMachine.java

```diff
@@ -325,7 +326,7 @@ public List<Path> getSnapshotFiles(File latestSnapshotRootDir) {
           region.getDatabaseName(),
           region.getDataRegionId(),
           e);
-      return null;
+      return Collections.emptyList();
```

Comment:
You need to check for upper-level dependencies on null, such as changing `== null` to `isEmpty` or something like that

Reply:
Already removed